en/components/grids_templates/row_selection.md

```diff
@@ -258,9 +258,240 @@ public getSelectedRows(args) {
 }
 ```
 
-### Styling Guidelines TODO
+### Row selectors templating
+You can template header and row selectors in the `@@igSelector` and also access their contexts which provide useful functionality for different scenarios.
+
+#### igxRowSelector
+With the `igxRowSelector` directive you can bind to the `@@igSelector` row context and access the following members:
+
+##### index
```

Comment:
Frankly, no need to section these like that. It's really weird with property titles.
Just put the `selected` and `index` in a sentence and show the snippet with a link to the basic demo